src/app/app.component.spec.ts

```diff
@@ -1,6 +1,9 @@
 import { TestBed, async } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
+import {HeaderComponent} from "./header/header.component";
```

Comment:
```suggestion
import { HeaderComponent } from './header/header.component';
```
Non-consistent quotes and spaces before/after the brackets